pyproject.toml

```diff
@@ -16,6 +16,7 @@ dependencies = [
     "pykdtree",
     "pyorbital",
     "pyproj>=2.2",
+    "pyPublicDecompWT",
```

Comment:
I think this should be in the optional dependencies for seviri_l1b_hrit rather than here, right?

Reply:
Yes, this is correct. Note that the `seviri_l1b_hrit` functionality will no longer be available without explicitly installing the necessary dependencies. Therefore, the default functionalities after a plain `pip install satpy` will change slightly. Even for reading uncompressed HRIT files, the `pyPublicDecompWT` package will be required. From my perspective, this is not an issue.

Reply:
Ok, so I think we need to move that dependency to the optional dependency part.
But also, in order to keep backwards compatibility, we should do the import inside the `decompress` function. Otherwise, as you say, the library needs to be installed even when reading non-compressed data.